doc/content/sqa/v_and_v/materials/isoil/pressure_dependency.md

```diff
@@ -0,0 +1,139 @@
+
+!media media/materials/isoil/pressure_dependent_strength1.png
+       id=strength-comparison1
+       caption=The user-defined backbone curve and the engineering shear stress-strain curve in the ZX plane output by MASTODON during the initial loading cycle (from 3 seconds to 4 seconds) for for pressure dependent strangth Cases 1, 2, and 3.
+
+!media media/materials/isoil/pressure_dependent_strength2.png
+       id=strength-comparison2
+       caption=The user-defined backbone curve and the engineering shear stress-strain curve in the ZX plane output by MASTODON during the initial loading cycle (from 3 seconds to 4 seconds) for pressure dependent strength cases 2, 4, and 5.
+
+Since setting $a_{2}=1$ and $a_{1}=0$ causes the shear strength to increase exponentially, while setting $a_{2}=0$ and $a_{1}=1$ causes it to increase only linearly, [strength-comparison1] shows that Case 1 is significantly stronger than Case 2. Also, since setting $a_{0}=1$ while setting the others to zero causes the right-hand-side of [p-ref-strength] to be equal to unity, [strength-comparison1] shows that Case 3 is practically equivalent to the original backbone curve (the slight discrepancy arises from the presence of initial stresses, the tensile cutoff value, $p_{0}=-1$, and the application of the body force). Cases 4 and 5 are essentially Case 2 except with different $p_{ref}$ values, hence, Case 2 was compared against them. [strength-comparison2] shows that setting the reference pressure equal to the applied vertical pressure (Case 4) causes the shear strength to remain virtually unmodified, while setting it lower (Case 2) causes the shear strength to increase and setting it higher (Case 5) causes the shear strength to decrease. These observations made from each of the five cases are arithmetically consistent with [p-ref-strength].
+
+#### Complete Input File
+
+!listing test/tests/materials/isoil/HYS_pressure_dependent_strength.i
+
+*Some modifications to this input file were made to produce the results presented on this page (i.e., the different parameter values used for each of the five cases). Please refer to [this section](#strength) for a description of these changes.*
+
+### Pressure Dependent Stiffness id=stiffness
+
+The three cases described in [stiffness-cases] were used to investigate the behavior of [p-ref-modulus] with different values for the parameter $b$. In all cases, the pressure dependent strength parameters were set to the following: $a_{0}=1$, $a_{1}=0$, and $a_{2}=0$, therefore, the yield strength was not modified, and $p_{ref}$ was set to 6072.86 Pa.
+
+*+Warning+*: If all three pressure dependent dependent strength parameters are set to zero in the input file, the solver will struggle to converge. One can simply set $a_{0}=1$ and the others to zero if they truly want to ignore shear strength dependency.
```

Comment:
If all three are zero, we need to throw an error, if we aren't throwing one already. Please add that to the code, if you can -- both in ComputeISoilStress.C and ISoilAction.C